modules/jaeger-config-ingester.adoc

```diff
@@ -8,31 +8,25 @@ This REFERENCE module included in the following assemblies:
 
 Ingester is a service that reads from a Kafka topic and writes to another storage backend (Elasticsearch).  If you are using the `allInOne` or `production` deployment strategies, you do not need to configure the Ingester service.
 
-.Jaeger Ingester parameters used by the Operator
-[options="header"]
-[cols="l, a, a"]
-|===
-|Parameter |Description |Values
-|ingester:
-  deadlockInterval:
-| Specifies the interval (in seconds or minutes) that the Ingester should wait for a message before terminating.
-The deadlock interval is disabled by default (set to 0), to avoid the Ingester being terminated when no messages arrive while the system is being initialized.
-|Minutes and seconds, for example, `1m0s`.  Default value is `0`.
-|===
-
-
 .Jaeger parameters passed to the Ingester
 [options="header"]
 [cols="l, a, a"]
 |===
 |Parameter |Description |Values
 |spec:
-  strategy: streaming
   ingester:
     options: {}
 |Configuration options that define the Ingester service.
 |
 
+|spec:
+  ingester:
```

Comment:
formats should be consistent, I think we can remove the `spec: ingester:` from this table cell

![image](https://user-images.githubusercontent.com/1004403/113332154-6ebbf800-933e-11eb-97d3-69aa23019ab0.png)